### ISSUE TITLE
Single Note Gallery Popup Mode?

### ISSUE THREAD

**Issue by denemier** [Darakah/obsidian-gallery#10]
Your Gallery Mode is Like a File Browser for directory.
Can you also have a option when we click a button,
and then it shows us all media files in a popup gallery that are related in my currently open note file in the editor?

Why this so important is, think for a second that we have small images shown inside a table, doesn't matter how small they are but we can view them in a gallery format quickly and larger, and we can make image sizes in our notes smaller for better scrolling space advantage and view them when we need.

**Comment by Darakah** [comment]
You want to be able to click on normal embedded images and have them focused like when using the gallery? 
Not sure I get what you want.

**Comment by denemier** [comment]
yes exactly popup gallery for embedded images in a note file, if shoes note has 17 images and i click a embedded image in preview modeiit shows 17 images on the same page, if it has 5 total then all 5 of them, this is just a enhancement and your decision.

And notice that there are some css hacks to build popup image zoomer like https://forum.obsidian.md/t/enlarge-image/9184
but it would be nice to see:
- 1 large zoomed image, and others below smaller like
![image](https://user-images.githubusercontent.com/63566495/115331586-e0af8200-a131-11eb-91dd-ebc5608e03dd.png)
- and switch with arrow keys or mouse wheel
but this is another direction decision for you, because
this way it would be complete gallery solution
- media directory viewer
- media tagger
- in single note gallery support

But i also want to say, that this request isn't more important for me than the planned **search by tag** feature or **imageinfo file naming, duplication** problem
